nifi-nar-bundles/nifi-hive-bundle/nifi-hive3-processors/src/main/java/org/apache/hadoop/hive/ql/io/orc/NiFiOrcUtils.java

```diff
@@ -104,6 +105,10 @@ public static Object convertToORCObject(TypeInfo typeInfo, Object o, final boole
             if (o instanceof Double) {
                 return new DoubleWritable((double) o);
             }
+            // Map BigDecimal to a Double type - this should be improved to map to Hive Decimal type
+            if (o instanceof BigDecimal) {
```

Comment:
There were some runtime issues with unit tests related to Orc. @mattyb149 @bbende @ijokarumawak do y'all have any insight into this ORC conversion?